### ISSUE TITLE
Release spring-batch-neo4j version 1.0.0

### ISSUE THREAD

**Issue by scordio** [spring-projects/spring-batch-extensions#157]
I prepared a [draft](https://github.com/spring-projects/spring-batch-extensions/releases) for the release notes.

**Comment by scordio** [comment]
Indeed, @meistermeier – I wasn't thinking about checking if the SDN version was the latest available.

To guarantee interoperability, I let Spring Boot drive all the dependency versions in the Notion module:

https://github.com/spring-projects/spring-batch-extensions/blob/0b9b9ad400ddb8bc5a4a944bba652b111926c387/spring-batch-notion/pom.xml#L5-L10

We may not want to refactor the Neo4j POM right now, but I agree we should bring SDN to the latest.

Is it a simple version increase, or is there more to consider when upgrading from 7.2 to 7.3?

**Comment by scordio** [comment]
Raised #158.

**Comment by meistermeier** [comment]
Yes, 7.3 can be used as a drop-in replacement for 7.2. No changes required.

**Comment by scordio** [comment]
We might want to wait for #159 before releasing.

**Comment by scordio** [comment]
@fmbenhassine I think we are ready to release the Neo4j module 🙂

**Comment by fmbenhassine** [comment]
I acknowledge this request to release the neo4j module. I am just waiting for an update from our Infra team to be able to proceed with the release to Maven Central. I will update this ticket once the release is done. Thank you for your comprehension.

**Comment by scordio** [comment]
Hi @fmbenhassine, I just wanted to highlight that [OSSRH will reach end-of-life](https://central.sonatype.org/news/20250326_ossrh_sunset/) on June 30th, in case this will impact the release process of the Spring Batch extensions.

**Comment by fmbenhassine** [comment]
Working on this right now, I will post a message here when done.

**Comment by fmbenhassine** [comment]
Version 1.0.0 is in Maven Central: https://central.sonatype.com/artifact/org.springframework.batch.extensions/spring-batch-neo4j

Congrats on the release!

PS: I did not update the next development version to let you decide which version goes next.

**Comment by fmbenhassine** [comment]
Thank you! I will let every module lead publishes the release notes as they know better than I do the content of each release.
